plonk-wasm/src/rayon.rs

```diff
@@ -13,20 +13,22 @@
 
 use js_sys::Promise;
 use spmc::{channel, Receiver, Sender};
+use std::{cell::RefCell, thread_local};
 use wasm_bindgen::prelude::*;
 
 #[cfg(feature = "nodejs")]
 use js_sys::JsString;
 
-static mut THREAD_POOL: Option<rayon::ThreadPool> = None;
+thread_local! {
```

Comment:
Yeah come to think of that I _really_ don't understand the use of `thread_local!` here. It seems that this thing should be accessible only in a single thread manner, so maybe what you want is a static `Mutex`.